frontend/src/components/GovtMasthead/GovtMasthead.tsx

```diff
@@ -21,36 +21,71 @@ import { GovtMastheadItem } from './GovtMastheadItem'
 export const GovtMasthead = (): JSX.Element => {
   const { isOpen, onToggle } = useDisclosure()
 
+  // Custom function for collapsing/expanding the header,
+  // because in mobile you tap the whole header, on desktop only on the link
+  const isMobile =
+    useBreakpointValue({
+      base: true,
+      xs: true,
+      md: false,
+    }) ?? true
+
+  interface HeaderBarProps {
+    isMobile: boolean
+    children: React.ReactNode
+  }
+
+  interface HowToIdentifyProps {
+    isMobile: boolean
+  }
+
+  const HeaderBar = ({ isMobile, children }: HeaderBarProps): JSX.Element => {
+    const styleProps = {
+      bg: 'neutral.200',
+      py: { base: '0.5rem', md: '0.375rem' },
+      px: { base: '1.5rem', md: '1.75rem', lg: '2rem' },
+      textStyle: { base: 'legal', md: 'caption-2' },
+      display: 'flex',
+      width: '100%',
+    }
+    return (
+      <Flex {...styleProps} {...(isMobile ? { onClick: onToggle } : {})}>
```

Comment:
i would suggest a semantic HTML tag instead of calling onClick for Flex (since that is not accessible):

```tsx
const HeaderBar = ({isMobile}) => {
  const styleProps = {...}
  if (isMobile) {
    return <chakra.button {...styleProps} onClick={onToggle}>{children}</chakra.button>
  }
  
  // Non-mobile
  return <Flex {...styleProps}>{children}</Flex>
}

```

Reply:
thanks!